src/main/java/io/github/hypixel_api_wrapper/wrapper/util/GuildLevelingUtil.java

```diff
@@ -0,0 +1,159 @@
+package io.github.hypixel_api_wrapper.wrapper.util;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+// source code: https://github.com/HypixelDev/PublicAPI/blob/master/hypixel-api-core/src/main/java/net/hypixel/api/util/IGuildLeveling.java
+public class GuildLevelingUtil {
```

Comment:
This class's methods should be changed to public since it's no longer an interface